db/migrate/20171115045045_add_status_to_answers.rb

```diff
@@ -1,4 +1,4 @@
-class AddStatusEnumToAnswer < ActiveRecord::Migration[5.1]
+class AddStatusToAnswers < ActiveRecord::Migration[5.1]
```

Comment:
why did you change this? this migration is already included in master so everyone already has it, which means it shouldn't be changed anymore